src/editor/editor.js

```diff
@@ -116,7 +124,10 @@ export default class Editor {
 			.then( () => this.fire( 'pluginsReady' ) );
 
 		function loadPlugins() {
-			return that.plugins.load( config.get( 'plugins' ) || [] );
+			const pluginsFromConfig = config.get( 'plugins' ) || [];
+			const pluginsToLoad = pluginsFromConfig.concat( that._availablePlugins );
```

Comment:
This doesn't follow: https://github.com/ckeditor/ckeditor5-core/issues/67 

```js
this.config.define( 'plugins', availablePlugins )
```

was supposed to be done.